extension/content/components/recipes/details/DetailsHeader.js

```diff
@@ -85,7 +99,35 @@ export default function DetailsHeader() {
       console.warn(err.message, err.data);
       Alert.error(`An Error Occurred: ${JSON.stringify(err.message)}`, 5000);
     } finally {
-      setIsButtonLoading(false);
+      removeButtonLoading("disable");
+    }
+  };
+
+  const handlePauseClick = async () => {
+    addButtonLoading("pause");
+    try {
+      const updatedData = await normandyApi.patchRecipe(data.recipe.id, {
+        comment: "One-click pause",
+        arguments: {
+          ...data.arguments,
```

Comment:
Do we do this because the merge of dict keys is not «recursive»? Maybe a comment would be useful